playbooks/hosts/amc_production/group_vars/edx

```diff
@@ -0,0 +1,204 @@
+  ENABLE_INSTRUCTOR_ANALYTICS: false  # default: false
+  SUBDOMAIN_BRANDING: false  # default: false
+  SUBDOMAIN_COURSE_LISTINGS: false  # default: false
+  PREVIEW_LMS_BASE: "{{ EDXAPP_PREVIEW_LMS_BASE }}"
+  ENABLE_S3_GRADE_DOWNLOADS: true  # default: true
+  USE_CUSTOM_THEME: "{{ edxapp_use_custom_theme }}"
+  ENABLE_MKTG_SITE: "{{ EDXAPP_ENABLE_MKTG_SITE }}"  # default: false
+  AUTOMATIC_AUTH_FOR_TESTING: "{{ EDXAPP_ENABLE_AUTO_AUTH }}"  # default: false
+  ENABLE_THIRD_PARTY_AUTH: "{{ EDXAPP_ENABLE_THIRD_PARTY_AUTH }}" # default: false
+  ENABLE_VIDEO_UPLOAD_PIPELINE: false  # default: false
+  ENABLE_DISCUSSION_HOME_PANEL: true  # default: true
+  ENABLE_CONTENT_LIBRARIES: true  # default: true
+  #search features in CMS
+  ENABLE_COURSEWARE_INDEX: true  # default: false
+  ENABLE_LIBRARY_INDEX: true  # default: false
+  #search features in LMS
+  ENABLE_COURSEWARE_SEARCH: true  # default: false
+  ENABLE_DASHBOARD_SEARCH: true  # default: false
+  ENABLE_COURSE_DISCOVERY: false # default: false
+  #changes for matej's design
+  #THEME_FOLDER: ""
+  ENABLE_COMBINED_LOGIN_REGISTRATION: true  # default: true
+  DISABLE_COURSE_CREATION: true
+
+EDXAPP_PLATFORM_NAME: "Appsembler Academy"
+COMMON_ENVIRONMENT: "default-env"
+COMMON_DEPLOYMENT: "default-staging"
+
+## Theme
+edxapp_use_custom_theme: false # false to disable & use std. theme
+edxapp_theme_name: "default"
+edxapp_theme_source_repo: "https://github.com/appsembler/edx-theme.git"
+edxapp_theme_version: "master" #branch
+
+#disable analytics link in instructor dashboard
+#EDXAPP_ANALYTICS_DASHBOARD_URL: ""  # if enabled: http://insights.examples.com
+
+## NGINX Config
+NGINX_ENABLE_SSL: true
+NGINX_REDIRECT_TO_HTTPS: true
+NGINX_SSL_CERTIFICATE: "/Users/bezidejni/Documents/projekti/appsembler/appsembler.crt"
+NGINX_SSL_KEY: "/Users/bezidejni/Documents/projekti/appsembler/appsembler.key"
+NGINX_SET_X_FORWARDED_HEADERS: True #set to True if not behind load balancer
+EDXAPP_LMS_NGINX_PORT: 80
+EDXAPP_LMS_SSL_NGINX_PORT: 443
+EDXAPP_LMS_PREVIEW_NGINX_PORT: 80
+EDXAPP_CMS_NGINX_PORT: 80
+EDXAPP_CMS_SSL_NGINX_PORT: 443
+CMS_HOSTNAME: '~^((stage|prod)-)?studio.*' #for redirect within nginx
+
+nginx_sites:
+  - certs
+  - cms
+  - lms
+  - forum
+  - xqueue
+nginx_default_sites:
+  - lms
+#COMMON_HTPASSWD_USER: "user"
+#COMMON_HTPASSWD_PASS: "password"
+COMMON_ENABLE_BASIC_AUTH: False
+
+NGINX_ROBOT_RULES:
+  - { agent: '*', disallow: '/' }
+
+## Base
+EDXAPP_LMS_BASE: "academy.appsembler.com"
+EDXAPP_PREVIEW_LMS_BASE: "academy-preview.appsembler.com"
+EDXAPP_CMS_BASE: "academy-studio.appsembler.com"
+CMS_HOSTNAME: "academy-studio.appsembler.com"
+
+## Site names
+EDXAPP_SITE_NAME: "academy.appsembler.com"
+EDXAPP_LMS_SITE_NAME: "academy.appsembler.com"
+EDXAPP_CMS_SITE_NAME: "academy-studio.appsembler.com"
+
+## Social media
+EDXAPP_PLATFORM_FACEBOOK_ACCOUNT: "http://www.facebook.com/accountName"
+EDXAPP_PLATFORM_TWITTER_ACCOUNT: "@twitterHandle"
+
+## Email addresses
```

Comment:
I'd say for now, just change @example.com to @appsembler.com since all wildcard addresses will go to my email address.